sample_metadata/parser/generic_parser.py

```diff
@@ -75,6 +77,19 @@
 SUPPORTED_VARIANT_TYPES = Literal['gvcf', 'vcf']
 
 
+class SequenceMetaGroup(BaseModel):
```

Comment:
This doesn't need to inherit from pydantic.BaseModel, as we're not serialising this model it right?